### PR TITLE
docs:  update policy around private vars 

### PR DESCRIPTION
**Pull request by malikchaya2**

### CODE REVIEW

File: docs/Project-Configuration/Project-and-Distro-Settings.md [hunk 1]
```diff
@@ -127,7 +127,8 @@ file via an expansion.
 Options:
 
 -   Checking **private** makes the variable redacted so the value won't
-    be visible on the projects page or by API routes.
+    be visible on the projects page or by API routes. Per  
+    [evergreen policy](https://mongodb.stackenterprise.co/questions/1232), private variables cannot be retrieved.
```
Comment (ablack12) [inline docs/Project-Configuration/Project-and-Distro-Settings.md:131]: minor nit, we should capitalize Evergreen

Comment (malikchaya2) [reply docs/Project-Configuration/Project-and-Distro-Settings.md]: thanks, fixed

**Review by ablack12 — COMMENTED** [review]
Thanks! I think the change looks good but the PR title should still reflect guidelines:
```
docs: <updated xyz>
```

**Review by ablack12 — APPROVED** [review]
oops meant to publish that all as an approve, sorry 🤦‍♀️

**Pull request merged**